### PR TITLE
Day 1

### PR DESCRIPTION
**Pull request by Oscar-Espinoza**
Home and about views are done, all the styles are added and the functionality, also the readme file with my loom video doing a presentation on the website there are no linter errors so I'm just waiting for your kind review, hope you have a good time 😄

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+        <i class="fa-regular fa-circle-xmark close-btn menu-btn" id="close-btn"></i>
+        <img src='media/event-logo.png' alt='logo' class='nav-logo' id='nav-logo'>
+        <ul class="nav-links">
+          <li class="nav-link"><a href="#" id="about-link">About</a></li>
+          <li class="nav-link"><a href="#">Program</a></li>
+          <li class="nav-link"><a href="#">Join</a></li>
+          <li class="nav-link"><a href="#">Sponsor</a></li>
+          <li class="nav-link"><a href="#">News</a></li>
+          <li class="nav-link"><a href="#">YWCC Campaign</a></li>
+        </ul>
```
Comment (Munsa1) [inline index.html]: - [x] Kindly implement the Orange button to the left as it is in the given design, also implement the pressed button effect. If you look closeley you will see that the page on which the user is has a highlighted color to its corresponding link in the navigation.
#### Note that this applies fr the about page as well

Your design
image

Given design
image

File: index.html [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+
+      <div class="more-btn active" id="more-btn">
+        <p>MORE</p><i class="fa-solid fa-chevron-down"></i>
+      </div>
+    </section>
+  </div>
+
+  <div class="about-page" id="about-page">
+
+    <section class="about">
```
Comment (Munsa1) [inline index.html]: - [x] The requirement is that you have to create two seperate html files, the index.html and the about.html respectively, kindly do so.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,29 @@
...
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page.
+
+Show your support
+Give a ⭐️ if you like this project!
+
+## Watch my walkthrough video for the website
+
+Loom video link:
```
Comment (Munsa1) [inline README.md]: - [x] Kindly use the new Microverse template for the readme file as this is the latest requirement. [Follow this link](https://github.com/microverseinc/readme-template/blob/master/README.md)

File: index.html [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+
+  <div class="home-page active" id="home-page">
+    <section class="home">
+      <h1 class="p-title">YEARLY COMEDIANS WORLD CONFERENCE</h1>
+      <p class="description">A joyful reunion of the most popular living comedians from all over the globe, to celebrate comedy, taking place this December in Spain.</p>
+      <div class="date-and-place">
+        <p class="date">2022.12.15(THU) - 16(FRI)</p>
+        <p class="place">@ Camp Nou, C. d'Arístides Maillol, 12, 08028 Barcelona, Spain</p>
+      </div>
+    </section>
```
Comment (Munsa1) [inline index.html]: - [x] This section has a subtitle at the top followed by the man title in the original design. kindly implement it in your design as well as this is a requirement.

Your design
image

Given design
image

File: index.html [hunk 5]
```diff
@@ -0,0 +1,158 @@
...
+      </div>
+  
+      <ul class="comedians" id='comedians-list'>
+      </ul>
+
+      <div class="more-btn active" id="more-btn">
+        <p>MORE</p><i class="fa-solid fa-chevron-down"></i>
+      </div>
+    </section>
+  </div>
```
Comment (Munsa1) [inline index.html:113]: - [x] The partners and footer section have not been implemented in the index.html file and this includes the mobile view, kindy add them according to the given design.
image

#### This applies for the about page as well

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @Oscar-Espinoza, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

Great video presentation, it was professional, clear and straight to the point


## Suggested changes
_Check the comments under the review._
- [x] The Project has not been deployed using GIthub Pages, kindly do so and provide the link in the readme file.
image

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: home.html [hunk 6]
```diff
@@ -0,0 +1,132 @@
...
+  <div class="home-page active" id="home-page">
+    <section class="home">
+      <h2 class="subtitle-home">"Hello! Sharing World"</h2>
+      <h1 class="p-title">YEARLY WORLD COMEDIANS CONFERENCE</h1>
+      <p class="description">A joyful reunion of the most popular living comedians from all over the globe, to celebrate comedy, taking place this December in Spain.</p>
+      <div class="date-and-place">
+        <p class="date">2022.12.15(THU) - 16(FRI)</p>
+        <p class="place">@ Camp Nou, C. d'Arístides Maillol, 12, 08028 Barcelona, Spain</p>
+      </div>
+    </section>
```
Comment (aimalamiri) [inline home.html]: It looks great, but it's highly recommended to follow the original design idea as much as possible. Especially the colors, typography, and sizing. As you can see in the header the text size is much smaller than the original one, and the color is also not similar to the one in the original design **(Idea: Using the CSS [gradients](https://developer.mozilla.org/en-US/docs/Web/CSS/gradient/linear-gradient) can be helpful)**. Please make sure to increase the text size and use a similar color.


Yours
image

Original
image

File: home.html [hunk 7]
```diff
@@ -0,0 +1,132 @@
...
+      </div>
+  
+      <ul class="activities">
+  
+        <li class="activity">
+          <img src='/media/public-speaker-icon.png' alt="" class="icon">
+          <h3 class="title">Lecturee</h3>
+          <p class="description">Lorem ipsum dolor sit amet consectetur adipisicing elit. Libero esse eveniet est.</p>
+        </li>
+
```
Comment (aimalamiri) [inline home.html]: The design is implemented very well, but you can improve it by increasing its font size here.

Yours
image

Original
image

File: home.html [hunk 8]
```diff
@@ -0,0 +1,132 @@
...
+          <h3 class="title">Lecturee</h3>
+          <p class="description">Lorem ipsum dolor sit amet consectetur adipisicing elit. Libero esse eveniet est.</p>
+        </li>
+  
+        <li class="activity">
+          <img src="/media/board-icon.png" alt="" class="icon">
+          <h3 class="title">Conference 
+            Exhibition</h3>
+          <p class="description">Lorem ipsum dolor sit amet, consectetur adipisicing elit.</p>
+        </li>
```
Comment (aimalamiri) [inline home.html]: Adding a hover effect will make it somewhat interactive. As you can see in the original design you can implement the hover effect by adding a white bold border to these cards. Please make sure to implement this.

image

File: app.js [hunk 9]
```diff
@@ -0,0 +1,92 @@
...
+    comedianElement.classList.add('comedian');
+    comedianElement.innerHTML = `
+      <div class='photo'><img src='media/${comedian.photoUrl}'></div>
+      <div class='information'>
+        <h3 class='name'>${comedian.name}</h3>
+        <p class='description'>${comedian.description}</p>
+        <hr>
+        <p class='background'>${comedian.background}</p>
+      </div>
+      `;
```
Comment (aimalamiri) [inline app.js:59]: I like the way you implemented the javascript for generating the speakers. But it seems you can improve the design by following the original design idea. So for this please make sure to add the background picture for the speaker image, and also increase its size a little bit to make it look much more similar to the one in the original design..

image

File: about.html [hunk 10]
```diff
@@ -0,0 +1,98 @@
...
+  </header>
+
+  <section class="about">
+    <h1 class="p-title">YEARLY WORLDWIDE COMEDIANS CONFERENCE</h1>
+    <p class="about-description">Lorem ipsum dolor sit amet consectetur adipisicing elit. Quae porro impedit quod omnis repellendus amet consequatur iste aspernatur dolorem! Quaerat obcaecati excepturi recusandae molestias numquam.</p>
+    <div class="contact">
+      <p class="contact-description">Please contact us per Email for any further questions about YW Comedians Conference 2022</p>
+      <p class="email">ywcc2022@ywspain.com</p>
+    </div>
+  </section>
```
Comment (aimalamiri) [inline about.html]: You're almost there, but you still have some opportunities to improve by following the sizing from the original design idea. 

Here I'd highly recommend increasing the text size.

Yours
image

Original
image

File: README.md [hunk 11]
```diff
@@ -1,2 +1,97 @@
...
+
+- **You can use the navbar to move through the different pages of the site**
+- **It is responsive for desktop and mobile screen sizes**
+- **Some parts are dinamically generated with Javascript**
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://www.loom.com/share/4685dac778c7469fb5d9b3a97d199be6)
```
Comment (aimalamiri) [inline README.md]: The live demo link must be the link to your deployed web page, not to your presentation video. Kindly make sure to update this.

File: README.md [hunk 12]
```diff
@@ -1,2 +1,97 @@
...
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project feel free to follow me on github or send me a friend request on Linkedin so we can connect
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse for encourage me to do this project.
```
Comment (aimalamiri) [inline README.md]: It's required to mention the original design idea owner. In our case, it's [Cindy](https://www.behance.net/adagio07). Please make sure to update your readme file with proper credits.

**Review by aimalamiri — CHANGES_REQUESTED** [review]
Hi @Oscar-Espinoza, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

AlmostThereReachGIF


 ## Highlights

- The pages are designed very well and looks fantastic ✔️ ⭐
- Good job for using the right GitHub Flow 💯
- Awesome file organization 🗃️
- Commits are descriptive and meaningful 👍
- Linters are properly set up and passing 🟢 👍

 ## Required Changes ♻️

You did a great job, but you still have some points that can be improved which are mentioned in the bellow comments, but in here I'd like to mention that your project is not deployed properly. Because when I visited the deployment link I was able to see the Readme file not your web page. The possible reason for this in my opinion is that you renamed the `index.html` file to `home.html` page. Please make it `index.html` again so the GitHub can show the proper page.

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 13]
```diff
@@ -1,2 +1,104 @@
...
+
+- **You can use the navbar to move through the different pages of the site**
+- **It is responsive for desktop and mobile screen sizes**
+- **Some parts are dinamically generated with Javascript**
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://oscar-espinoza.github.io/CC-Global-Summit/)
```
Comment (jerryowusu) [inline README.md:56]: [OPTIONAL] Well done for successfully deploying the project on github. :+1:  However, you need to ensure that everything works fine as it's suppose to be. Currently the images under the main program section are not displaying, kindly check and fix it. :pray:

**Review by jerryowusu — APPROVED** [review]
Hi @Oscar-Espinoza , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

clap approve


## Highlights
Neat and clean code :heavy_check_mark: 
All linters are passing :heavy_check_mark: 
Followed GitHub flow :heavy_check_mark: 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**